pyfar/classes/coordinates.py

```diff
@@ -42,7 +42,8 @@ def __init__(
         domain use :py:func:`from_spherical_colatitude`,
         :py:func:`from_spherical_elevation`, :py:func:`from_spherical_front`,
         :py:func:`from_spherical_side`, or :py:func:`from_cylindrical`
-        instead.
+        instead. For conversions from or into degree
+        use :py:func:`pf.deg2rad` and :py:func:`pf.rad2deg`.
```

Comment:
Did you check how this looks in the docs? We usually use, which display only the function name:
```suggestion
        use :py:func:`~pf.deg2rad` and :py:func:`~pf.rad2deg`.
```

Reply:
not sure, it did not change anything in the doc, so I left it. But I fixed the links in the doc